### ISSUE TITLE
Ion Status Bars lua error

### ISSUE THREAD

**Issue by teh-fixerer** [SLOKnightfall/Ion-Core#39]
I get this error with Ion status bars:
Message: Interface\AddOns\Ion-StatusBars\Ion-StatusBars.lua:1064: Usage: PlaySound(soundKitID, optional["SFX","Music","Ambience", "Dialog" or "Master"], [forceNoDuplicates, runFinishCallback])
Time: 08/31/17 06:29:17
Count: 1
Stack: Interface\AddOns\Ion-StatusBars\Ion-StatusBars.lua:1064: Usage: PlaySound(soundKitID, optional["SFX","Music","Ambience", "Dialog" or "Master"], [forceNoDuplicates, runFinishCallback])
[C]: ?
[C]: in function `PlaySound'
Interface\AddOns\Ion-StatusBars\Ion-StatusBars.lua:1064: in function <Interface\AddOns\Ion-StatusBars\Ion-StatusBars.lua:1045>

Locals:

**Comment by brittyazel** [comment]
Yes, this has been fixed in my fork of this add-on. I'm just waiting on SLOKnightfall to incorporate my changes

**Comment by AlexFolland** [comment]
While you wait for the merge, you can use https://wow.curseforge.com/projects/playsoundfixer to prevent all your addons from fully breaking when they encounter broken PlaySound calls.

**Comment by teh-fixerer** [comment]
@brittyazel 
I just realized there was another lua error in status bars:
Message: Interface\AddOns\Ion-StatusBars\Ion-StatusBars.lua:362: attempt to index local 'line' (a boolean value)
Time: 09/01/17 06:45:16
Count: 2
Stack: Interface\AddOns\Ion-StatusBars\Ion-StatusBars.lua:362: attempt to index local 'line' (a boolean value)
[C]: ?
Interface\AddOns\Ion-StatusBars\Ion-StatusBars.lua:362: in function <Interface\AddOns\Ion-StatusBars\Ion-StatusBars.lua:338>
Interface\AddOns\Ion-StatusBars\Ion-StatusBars.lua:373: in function <Interface\AddOns\Ion-StatusBars\Ion-StatusBars.lua:371>

Locals: 

Is this also a playsound issue?

**Comment by brittyazel** [comment]
That doesn't look like a playsound issue. Hrm, I'll have to look into that one

**Comment by brittyazel** [comment]
Can you try my build of Ion and submodules (installing it manually) from my fork and see if that issue is still present? It isn't immediately apparent to me what is causing this issue

**Comment by brittyazel** [comment]
I fixed this problem (and many more!) in the latest version. You can snag it over on Wowinterface while I figure out Curse. Also, please do me a favor and report bugs on my github, as that is what I'll be using for development for the time being

**Comment by teh-fixerer** [comment]
@brittyazel 
Awesome job - thanks so much!
